moveit_core/constraint_samplers/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 set(MOVEIT_LIB_NAME moveit_constraint_samplers)
 
+add_compile_options(-Wdeprecated-declarations)
```

Comment:
Please remove, deprecation warnings are intended to be treated as errors. If code is affected by deprecations, it needs to be fixed as well.